stdlib/typing_extensions.pyi

```diff
@@ -670,6 +687,12 @@ else:
         globals: Mapping[str, Any] | None = None,  # value types depend on the key
         locals: Mapping[str, Any] | None = None,  # value types depend on the key
         type_params: Iterable[TypeVar | ParamSpec | TypeVarTuple] | None = None,
-        format: Format = Format.VALUE,  # noqa: Y011
+        format: Format | None = None,
         _recursive_guard: Container[str] = ...,
     ) -> AnnotationForm: ...
+
+# PEP 661
+class Sentinel:
+    def __init__(self, name: str, repr: str | None = None) -> None: ...
+    def __or__(self, other: Any) -> UnionType: ...  # other can be any type form legal for unions
+    def __ror__(self, other: Any) -> UnionType: ...  # other can be any type form legal for unions
```

Comment:
hmm, we probably shouldn't have added these at runtime on Python <3.10 (we generally don't try to "backport PEP 604" for most typing_extensions features)

Reply:
Good point, it's an RC so we can still fix this.